composer.json

```diff
@@ -0,0 +1,6 @@
+{
+    "require-dev": {
+        "squizlabs/php_codesniffer": "*",
+        "magento/magento-coding-standard": "^6.0"
```

Comment:
I noticed we've also got this dependency listed in the [packages composer.json](https://github.com/degica/komoju-magento/pull/11/files#diff-bd85579e5fa504706f6dd9f2aa0b3f88065549792979253842a2edb65ae31f6cR14), except that's set as 5.0 or above. From memory, I added that just so we can the linter locally, so if we no longer need it we should remove it in this PR as well 😄

Reply:
Oh, I also wrote two scripts for us to running linting locally, [lint-check](https://github.com/degica/komoju-magento/blob/master/bin/lint-check#L1) and [lint-autofix](https://github.com/degica/komoju-magento/blob/master/bin/lint-autofix), if we could ensure those still work after this change (or we have another way to running the linting locally) that'd be fantastic 🙏

Reply:
Ah sorry, I realize now why this is here. I needed it for the CI so that I can run the sniffer there. So the package's composer.json is still separate from this one. lint-check and lint-autofix still work fine as it's not affected by this

Reply:
Ah I see 😄 . I'd really like to avoid having to keep two different dependency lists for dev and CI (because I _know_ I'm never going to remember to update them both 😛 ), could we get rid of one of them? I believe we need to keep the package specific composer.json to ensure the plugin functions correctly, so is it possible to configure CI to reference that composer.json file? Skimming the docs it looks like github workflows can set [a working directory](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepsrun) for the job, so I'm wondering if we could place this in the `src/app/code/Komoju/Payments/composer.json` file and then run the linting command from there?

Reply:
This one is tricky and for now I'm not sure what an optimal solution is. I actually tried this yesterday (running the lint from the directory you mentioned) but since that composer.json file is built assuming it's being run within a magento environment, it doesn't work outside of it. For example when it tries to autoload registration.php, you get something like this: 
```
vendor/bin/phpcs --standard=magento2 .
PHP Fatal error:  Uncaught Error: Class "Magento\Framework\Component\ComponentRegistrar" not found in [...]
```

Reply:
hmmmm, ok 😕 . Since I don't have a better solution let's leave it as is for now, and come back to this later, after v1.1 is deployed.